File: Java/Petshop.java
class Petshop{
    private int id;
    private String nama_produk;
    private int harga_produk;
    private int stok_produk;

    public Petshop(){
        this.id = 0;
        this.nama_produk = "";
        this.harga_produk = 0;
        this.stok_produk = 0;
    }

    public Petshop(int id, String nama_produk, int harga_produk, int stok_produk){
        this.id = id;
        this.nama_produk = nama_produk;
        this.harga_produk = harga_produk;
        this.stok_produk = stok_produk;
    }

    public int getId(){ return id; }
    public String getNamaProduk(){ return nama_produk; }
    public int getHargaProduk(){ return harga_produk; }
    public int getStokProduk(){ return stok_produk; }

    public void setId(int id){ this.id = id; }
    public void setNamaProduk(String nama_produk){ this.nama_produk = nama_produk; }
    public void setHargaProduk(int harga_produk){ this.harga_produk = harga_produk; }
    public void setStokProduk(int stok_produk){ this.stok_produk = stok_produk; }

    public int idLength(){ return String.valueOf(id).length(); }
    public int namaProdukLength(){ return nama_produk.length(); }
    public int hargaProdukLength(){ return String.valueOf(harga_produk).length(); }
    public int stokProdukLength(){ return String.valueOf(stok_produk).length(); }

}

File: Python/Petshop.py
class Petshop:
    __id = 0
    __nama_produk = ""
    __harga_produk = 0
    __stok_produk = 0

    def __init__(self, id, nama_produk, harga_produk, stok_produk):
        self._id = id
        self._nama_produk = nama_produk
        self._harga_produk = harga_produk
        self._stok_produk = stok_produk
    
    def getId(self):
        return self._id
    
    def getNamaProduk(self):
        return self._nama_produk
    
    def getHargaProduk(self):
        return self._harga_produk
    
    def getStokProduk(self):
        return self._stok_produk
    
    def setId(self, id):
        self._id = id

    def setNamaProduk(self, nama_produk):
        self._nama_produk = nama_produk

    def setHargaProduk(self, harga_produk):
        self._harga_produk = harga_produk

    def setStokProduk(self, stok_produk):
        self._stok_produk = stok_produk

    def idLength(self):
        return len(str(self._id))

    def namaProdukLength(self):
        return len(self._nama_produk)
    
    def hargaProdukLength(self):
        return len(str(self._harga_produk))
    
    def stokProdukLength(self):
        return len(str(self._stok_produk))
